scripts/package/package-debian.sh

```diff
@@ -171,6 +171,9 @@ copy_files_to_debian_layout(){
 
     # Copy Manpages
     cp -a "$MANPAGE_DIR/sdk/." "$PACKAGE_LAYOUT_DIR/docs"
+
+    # Copy postinstall
+    cp "$PACKAGING_ROOT/postinst" "$PACKAGE_LAYOUT_DIR/$"
```

Comment:
This is fairly different than what we discussed, and this will end up placing the script in `/` (root of the filesystem)

What about copying the `postinst` script to a new directory called `debian` rather than `$` which will allow for additional customization rather than this special case only?